cmd/version.go

```diff
@@ -27,6 +28,7 @@ func versionFunc(cmd *cobra.Command, args []string) error {
 	fmt.Println(packageName + ", " + Version)
 	fmt.Println("running on...")
 	fmt.Println(runtime.GOOS + ", " + runtime.Version())
+	fmt.Println("Git Info: " + GitCommit)
```

Comment:
I think there is still a bit of logic missing as `GitCommit wouldn't be set to none etc.`

Reply:
Happy to answer any specific questions, where you think something is still unclear or missing.

Reply:
I'm not 100% clear on the usage of the command: will there be different outputs based on whether you're running `version` on a dev build or a release build? I think that's what you were mentioning [here](https://github.com/resumic/schema/issues/76#issuecomment-479514315), but I just wanted to clarify that.

Reply:
A release build would show the exact version and details.
A dev build adds "-dev" to the version.
But probably better described in the section you linked.

Reply:
Adding this functionality and formatting in the next commit!

Reply:
Just added changes for this; the Makefile has 3 rules: `build`, `dev`, and `release`, which when respectively used to build the program result in the output specified.